PyInstaller/utils/hooks/django.py

```diff
@@ -74,7 +74,8 @@ def _remove_class(class_name):
         for cl in settings.AUTHENTICATION_BACKENDS:
             cl = _remove_class(cl)
             hiddenimports.append(cl)
-    if hasattr(settings, 'DEFAULT_FILE_STORAGE'):
+    # Deprecated since 4.2, may be None until it is removed
+    if getattr(settings, 'DEFAULT_FILE_STORAGE'):
```

Comment:
This is going to raise `AttributeError` if/when `DEFAULT_FILE_STORAGE` is really removed. 

Please implement proper `None` check instead.

Reply:
Sorry, I forgot to put `None` as the default value. It is fixed now.

Reply:
Indeed, but the attribute should be accessed only once. E.g.

```python
    cl = getattr(settings, 'DEFAULT_FILE_STORAGE', None)
    if cl:
        hiddenimports.append(_remove_class(cl))
```

Reply:
Makes sense, it is changed.